internal/provenance/open.go

```diff
@@ -71,6 +81,15 @@ func Open(conf *Config) (*Store, error) {
 		return openNewProvenanceStore(conf)
 	}
 
+	if conf.Disabled {
+		conf.Logger.Info("The provenance store is disabled on this node.")
+		disabledFlagPath := filepath.Join(conf.StoreDir, disabledFlag)
+		if _, err := os.Create(disabledFlagPath); err != nil {
```

Comment:
use `fileops.CreateFile()`

Reply:
The `fileops.CreateFile()` version has semantics of "create if does not exist", whereas the `os.Create()` is "create or truncate". I actually need the later version; since the file is just a flag, it saves me the need to check existence and create only if it does not exist.